content/about/contributing.md

```diff
@@ -5,20 +5,20 @@ source: https://github.com/cucumber/cucumber/wiki/Contributing/
 title: Contributing
 ---
 
-Please follow these guidelines if you want to contribute to Cucumber or help the cucumber team reproduce a problem.
+Please follow these guidelines if you want to contribute to Cucumber or help the Cucumber team reproduce a problem.
 
 ## You have found a bug
 
-Before you [get in touch](/about/get-in-touch) and file a ticket in the issue tracker, make sure you have tried the latest gem. Also try to \[\[install]] the latest code from Git. This way you don't waste the developers' time by reporting a bug that has already been fixed.
+Before you [get in touch](/about/get-in-touch) and open a new Issue, make sure you have tried the latest gem. Also try to \[\[install]] the latest code from Git. This way you don't waste the developers' time by reporting a bug that has already been fixed.
```

Comment:
"version" instead of "gem"? (to make it more 'generic', i.e. not Ruby specific)

Reply:
And also remove the [[]] around "install" (or there should be a link there).
Git should be GitHub.